src/Common/src/System/SR.cs

```diff
@@ -89,3 +97,20 @@ internal static string Format(string resourceFormat, object p1, object p2, objec
         }
     }
 }
+
+namespace System.Resources.Diagnostics
+{
+    [AttributeUsage(AttributeTargets.All)]
+    [ExcludeFromCodeCoverage]
+    internal sealed class ExcludeFromCodeCoverageAttribute : Attribute
```

Comment:
Do you worry that others will start depending on this ExcludeFromCodeCoverageAttribute that is being brought in from SR? If so does it make since to put it in its own file and just include it by default where SR.cs/vb is included?